CHANGELOG/7.5.md

```diff
@@ -1,5 +1,47 @@
 # 7.5 Changelog
 
+## [7.5.3]
+
+### General Cmdlet Updates and Fixes
+
+- Remove `OnDeserialized` and `Serializable` attributes from `Microsoft.Management.UI.Internal` project (#25831)
+- Make the interface `IDeepCloneable` internal (#25830)
+- Fix Out-GridView by replacing use of obsolete BinaryFormatter with custom implementation. (#25559)
```

Comment:
Move the Out-GridView fix to the top. That is the one that users care about the most.

```suggestion
- Fix Out-GridView by replacing use of obsolete BinaryFormatter with custom implementation. (#25559)
- Remove `OnDeserialized` and `Serializable` attributes from `Microsoft.Management.UI.Internal` project (#25831)
- Make the interface `IDeepCloneable` internal (#25830)
```